.github/workflows/playwright.yml

```diff
@@ -32,6 +32,11 @@ jobs:
         node-version: lts/*
     - name: Install dependencies
       run: npm ci
+    - name: Install .NET HTTPS Development Certificate
+      # if: matrix.os == 'ubuntu-latest'
+      run: |
+        dotnet dev-certs https --clean
```

Comment:
dev certs don't work on linux. Did you test this? https://github.com/tmds/linux-dev-certs

Reply:
Was just trying as linux cert was also not working in CI. Undoing this change

Reply:
Used Linux cert